Core/Src/main_task.c

```diff
@@ -390,13 +462,33 @@ static void run_upshift_sm(void)
 		// assisted shifting should be used)
```

Comment:
For the EXIT_GEAR state we wont want to use the reach_target_RPM_spark_cut() function, so you should delete it and its associated comment